File: CS/Views/SignInView.cs
namespace DXHtmlMessengerSample.Views {
    using System;
    using DevExpress.Utils.MVVM.UI;
    using DevExpress.XtraEditors;
    using DXHtmlMessengerSample.ViewModels;

    [ViewType(typeof(SignInViewModel))]
    public partial class SignInView : XtraUserControl {
        public SignInView() {
            InitializeComponent();
            if(!mvvmContext.IsDesignMode) {
                InitializeStyles();
                InitializeBindings();
            }
        }
        void InitializeStyles() {
            Styles.SignIn.Apply(signInView);
        }
        void InitializeBindings() {
            var fluent = mvvmContext.OfType<SignInViewModel>();
            fluent.SetObjectDataSourceBinding(signInBindingSource);
            // Bind sign-in buttons
            fluent.BindCommandToElement(signInView, "btnFacebook", x => x.SignInViaSocialNetwork);
            fluent.BindCommandToElement(signInView, "btnGoogle", x => x.SignInViaSocialNetwork);
            fluent.BindCommandToElement(signInView, "btnLinkedIn", x => x.SignInViaSocialNetwork);
            fluent.BindCommandToElement(signInView, "btnSignIn", x => x.SignIn);
        }
        sealed class Styles {
            public static Style SignIn = new SignInStyle();
            //
            sealed class SignInStyle : Style { }
        }
    }
}

File: VB/Views/SignInView.vb
Imports DevExpress.Utils.MVVM.UI
Imports DevExpress.XtraEditors
Imports DXHtmlMessengerSample.ViewModels

Namespace DXHtmlMessengerSample.Views
    <ViewType(GetType(SignInViewModel))>
    Partial Public Class SignInView
        Inherits XtraUserControl
        Public Sub New()
            InitializeComponent()
            If Not mvvmContext.IsDesignMode Then
                InitializeStyles()
                InitializeBindings()
            End If
        End Sub
        Sub InitializeStyles()
            Styles.SignIn.Apply(signInView)
        End Sub
        Sub InitializeBindings()
            Dim fluent = mvvmContext.OfType(Of SignInViewModel)()
            fluent.SetObjectDataSourceBinding(signInBindingSource)
            ' Bind sign-in buttons
            fluent.BindCommandToElement(signInView, "btnFacebook", Sub(x) x.SignInViaSocialNetwork())
            fluent.BindCommandToElement(signInView, "btnGoogle", Sub(x) x.SignInViaSocialNetwork())
            fluent.BindCommandToElement(signInView, "btnLinkedIn", Sub(x) x.SignInViaSocialNetwork())
            fluent.BindCommandToElement(signInView, "btnSignIn", Sub(x) x.SignIn())
        End Sub
        Private NotInheritable Class Styles
            Public Shared SignIn As Style = New SignInStyle()
            '
            Private NotInheritable Class SignInStyle
                Inherits Style
            End Class
        End Class
    End Class
End Namespace